srcs/Webserv.cpp

```diff
@@ -62,69 +62,85 @@ void	Webserv::setupSocket()
 	this->_listenFd = listenSocket.fd;
 }
 
+void	Webserv::handleClients()
+{
+	int fdSize;
+
+	fdSize = _fds.size();
+	for (int i = 1; i < fdSize; ++i) // i = 1, because we don't need to check the listening socket
+	{
+		std::cout << "Revents : " << _fds[i].revents << std::endl;
+		if (_fds[i].revents == 0)
+			continue ;
+
```

Comment:
When I run this and cancel disconnect the client it goes into an infinite loop, maybe a linux mac thing? 
Does this work on Linux (I'm assuming that's wha you are using at home)? Is that why this block of code is commented out?

Reply:
Yap, correct!